resources/js/Components/ComponentsPraktikans/LandingFooter.jsx

```diff
@@ -0,0 +1,5 @@
+export default function LandingFooter() {
+    return (
+        <footer className='fixed left-0 right-0 mt-10 bg-[#304D30] w-full h-12'></footer>
```

Comment:
you can add a name for `#304D30` in your `tailwind.config.js` without waiting for someone to add a customization name